homeassistant/components/local_calendar/calendar.py

```diff
@@ -84,12 +89,22 @@ def __init__(
         self._event: CalendarEvent | None = None
         self._attr_name = name
         self._attr_unique_id = unique_id
+        self._attr_color: RGBColor | None = color
```

Comment:
I think we can just write as:
```suggestion
        self._attr_color = color
```